README.md

```diff
@@ -264,6 +264,122 @@ Alby enforces [Conventional Commits Specification](https://www.conventionalcommi
 
 Alby uses [Weblate](https://weblate.org/en/) to manage translations for different locales. If you'd like to contribute, you can [add translations here](https://hosted.weblate.org/projects/getalby-lightning-browser-extension/getalby-lightning-browser-extension/).
 
+#### Rules for developers adding new i18n translation strings:
+
+[Not to be confused with language translations]
+
+**Use underscores instead of camelCase.**
+
+✅ Correct
+
+```JSON
+"pay_now": "Pay Now"
+```
+
+❌ Wrong
+
+```JSON
+"payNow": "Pay Now"
+```
+
+**To avoid confusion, we prefer indentation over underscores, i.e.**
+
+✅ Correct
+
+```JSON
+{
+  "blue": {
+    "label": "Blue"
+  }
+}
+```
+
+❌ Wrong
+
+```JSON
+{
+  "blue_label": "Blue",
+}
+```
+
+**Only indent strings when it is an input or has different attributes:**
+
+✅ Correct
+
+```JSON
+{
+  "edit": {
+    "title": "Edit Account",
+    "label": "Name",
+    "screen_reader": "Edit account name"
+  }
+}
+```
+
+❌ Wrong
+
+```JSON
+{
+  "edit": {
+    "title": "Edit Account"
+  }
+}
+```
+
+Correct way for this would be:
+
+```JSON
+{
+  "edit": "Edit Account"
+}
+```
+
+**Use title for heading tags and label for label tags.**
+
+**When the content is copy and you wish to divide it in parts, you can use numbers:**
+
+```JSON
+{
+  "enable": {
+    "request1": "Request approval for transactions",
+    "request2": "Request invoices and lightning information"
+  }
+}
+```
+
+**For button text, you can use common translations:**
```

Comment:
Maybe explain the 3 main entry-points we have currently above?
1. `translations`
1. `components`
1. `common`